api/cloud/v1/message.proto

```
@@ -1282,6 +1285,56 @@ message OTelMetricBatch {
+  int64 to_ts = 5;
+
+  // Optional filters for events/logs
+  string filter = 6;          // e.g. "source:kubernetes" for events
+  string sort = 7;            // e.g. "timestamp" for logs
+  int32 max_results = 8;      // limit returned items (0 = API default)
+
+  int32 timeout_seconds = 9;
+}
+
+enum DatadogQueryType {
+  DATADOG_QUERY_UNSPECIFIED = 0;
```

Buf (1.66.1)

[error] Enum value name "DATADOG_QUERY_UNSPECIFIED" should be prefixed with "DATADOG_QUERY_TYPE_".

(ENUM_VALUE_PREFIX)